src/hankel.jl

```diff
@@ -1,12 +1,18 @@
 # Hankel
-struct Hankel{T, V<:AbstractVector{T}} <: AbstractMatrix{T}
+struct Hankel{T, V<:AbstractVector{T}, H<:Integer, W<:Integer} <: AbstractMatrix{T}
```

Comment:
I think its more standard and clearer to do this as `HW<:Tuple{Integer,Integer}`.